
# material:
# velocity: m/ns
#   if air is 0.3 and material is 0 -> all of the energy is reflected
#   modeling two things:
#   - refraction direction
#   - how much percentage is transmitted or reflected? (fresnel)
materials:
  # air
  - velocity: 0.3
    ambient:  1.0
    diffuse:  0.0
    specular: 1.0
  # wall stone
  - velocity: 0.0
    ambient:  1.0
    diffuse:  0.0
    specular: 3000.0

material_id_air: 0

object_materials: [
  1, # 0: Big kaist / dcc mesh
]
